core/templates/pages/exploration-editor-page/editor-navigation/editor-navbar-breadcrumb.component.spec.ts

```diff
@@ -44,14 +43,13 @@ describe('Editor Navbar Breadcrumb directive', function() {
   beforeEach(angular.mock.inject(function($injector, $componentController) {
     $rootScope = $injector.get('$rootScope');
     ExplorationTitleService = $injector.get('ExplorationTitleService');
-    ExplorationPropertyService = $injector.get('ExplorationPropertyService');
     FocusManagerService = $injector.get('FocusManagerService');
     RouterService = $injector.get('RouterService');
 
     ExplorationTitleService.init('Exploration Title Example Very Long');
 
     spyOnProperty(
-      ExplorationPropertyService,
+      ExplorationTitleService,
```

Comment:
Why this change?

Reply:
I had inherited ExplorationPropertyService in ExplorationTitleService so now we have to create spy on ExplorationTitleService instead of ExplorationPropertyService as per my knowledge. if we don't replace this then last test case is failing.

Reply:
PTAL